composer.json

```diff
@@ -45,6 +45,7 @@
         "jms/di-extra-bundle": "~1.7",
         "sensio/generator-bundle": "~2.3|~3.0",
         "symfony/yaml": "~2.3|~3.0",
+        "phpunit/phpunit": "~4.3",
```

Comment:
In your travis:
`  - composer global require phpunit/phpunit:@stable fabpot/php-cs-fixer --no-update`
 So I think it will use this version